src/sdf_tools/utils.py

```diff
@@ -0,0 +1,96 @@
+import numpy as np
+
+import sdf_tools
+
+
+def compute_2d_sdf_and_gradient(grid_world, sdf_resolution, sdf_origin, frame='world'):
```

Comment:
I recommend changing the filename to `utils_2d.py` and this function name to `compute_sdf_and_gradient`.

Otherwise, add `_2d` to several of the functions below